### ISSUE TITLE
ActionChains gives an exception with IE 11 and selenium 3.4 

### ISSUE THREAD

**Issue by karimbeji** [SeleniumHQ/selenium#5286]
Hello community,
i had two problem usin selenium server 3.4 and IE 11.

First with the action chains it raises an exception while sending keys:        webdriver.ActionChains(driver).send_keys(Keys.ESCAPE).perform()

I will share the stack error

```
File "/usr/lib/python2.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 194, in check_response
    raise exception_class(message, screen, stacktrace)
WebDriverException: Message: sendKeysToActiveElement
Build info: version: '3.4.0', revision: 'unknown', time: 'unknown'
System info: host: 'TUNCWD0074', ip: '192.168.56.1', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_144'
Driver info: driver.version: RemoteWebDriver
Stacktrace:
    at org.openqa.selenium.remote.http.AbstractHttpCommandCodec.encode (AbstractHttpCommandCodec.java:220)
    at org.openqa.selenium.remote.http.AbstractHttpCommandCodec.encode (AbstractHttpCommandCodec.java:118)
    at org.openqa.selenium.remote.HttpCommandExecutor.execute (HttpCommandExecutor.java:158)
    at org.openqa.selenium.remote.service.DriverCommandExecutor.execute (DriverCommandExecutor.java:82)
    at org.openqa.selenium.remote.RemoteWebDriver.execute (RemoteWebDriver.java:637)
    at org.openqa.selenium.remote.RemoteExecuteMethod.execute (RemoteExecuteMethod.java:35)
    at org.openqa.selenium.remote.RemoteKeyboard.sendKeys (RemoteKeyboard.java:35)
    at org.openqa.selenium.support.events.internal.EventFiringKeyboard.sendKeys (EventFiringKeyboard.java:42)
    at org.openqa.selenium.remote.server.handler.interactions.SendKeyToActiveElement.call (SendKeyToActiveElement.java:56)
    at org.openqa.selenium.remote.server.handler.interactions.SendKeyToActiveElement.call (SendKeyToActiveElement.java:31)
    at java.util.concurrent.FutureTask.run (None:-1)
    at org.openqa.selenium.remote.server.DefaultSession$1.run (DefaultSession.java:176)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (None:-1)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (None:-1)
    at java.lang.Thread.run (None:-1)
```

  My second issue is: typing in forms inputs is baaaadly slooow (ps: i've tried several versions of selenium servers)

Thank you

**Comment by lmtierney** [comment]
Any reason you're using `3.4`? That's pretty dated. Starting with version `3.5` the IEDriverServer is W3C compliant.

As far as your typing being slow, please refer to http://jimevansmusic.blogspot.com/2014/09/screenshots-sendkeys-and-sixty-four.html

**Comment by karimbeji** [comment]
Because when use 3.8 or even other versions IE doesn't work. 
it only works with the 3.4 server version.

**Comment by lmtierney** [comment]
It works fine for me. Can you elaborate?

**Comment by barancev** [comment]
Is this issue still actual in the latest selenium version (that is 3.11 at the moment)?

It never will be fixed in 3.4. If you have other issues with the latest version please report them.

**Comment by karimbeji** [comment]
@barancev thank you so much ... it works now
